src/pages/Swap/Swap.tsx

```diff
@@ -1,13 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { TradeType } from 'tdex-sdk';
+import BigNumber from 'bignumber.js';
 import './Swap.css';
 import { handleInstall } from '../../utils';
+import { onSendAmountChange } from '../../utils/TedexSetup';
+
+enum Asset {
+  base = '5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225',
+  qoute = '9b3d7db7f93cae59e9407232b9864b54c21cba20b992d707fa45136fbfa9b8fa',
```

Comment:
typo `quote` not `qoute`. Replace in all code

Reply:
replaced with quote.